### ISSUE TITLE
How to get started / tutorial?

### ISSUE THREAD

**Issue by mariusa** [hapijs/jwt#15]
#### Context

* *node version*:  12
* *module version*: ?

#### What are you trying to achieve or the steps to reproduce ?

How to use this module? README or website https://hapi.dev/module/jwt/
doesn't have any get started tutorial.

Also, what's the difference between this & https://github.com/dwyl/hapi-auth-jwt2 ?
If this project isn't actively developed, simply use that as the official jwt module, to remove confusion.

**Comment by bolencki13** [comment]
The documentation doesn't seem complete on this plugin. Through some trial and error I was able to get my own implementation to work.

```js
...
// server is an instance of Hapi.Server
await server.register(Jwt)

server.auth.strategy('jwt', 'jwt', {
  keys: process.env.JWT_SECRET,
  verify: {
    // audience of the jwt
    aud: 'my-audience', // RFC 7519 says this is optional. This plugin requires it
    // issuer of the jwt
    iss: 'my-issuer', // RFC 7519 says this is optional. This plugin requires it
    // verify subject of jwt
    sub: 'my-sub' // RFC 7519 says this is optional. This plugin requires it
  },
  validate: (artifacts) => {
    const user = ... // get the user some how
    if (user) {
      return {
        isValid: true,
        credentials: {
          user: user
        },
        response: 'User authenticated successfully.',
      }
    } else {
      return {
        isValid: false,
        credentials: null,
        response: 'Invalid token.',
      }
    }
  }
})
...
```

Generate JWT
```js
const payload = {
  sub: 'my-sub', // needs to match definition above
  value: 'some value to auth user', // this is a custom key I used, it could be named anything. Value should be a way to authenticate the user
  aud: 'my-audience', // needs to match definition above
  iss: 'my-issuer' // needs to match definition above
}
const token = require('jsonwebtoken').sign(payload, process.env.JWT_SECRET)
```

Header to authenticate the request
```
Authorization: Bearer <token>
```

**Comment by cjihrig** [comment]
The documentation is indeed missing here. This module started off as closed source, and no one ever got around to writing the docs. Pull requests are welcome though!

**Comment by devinstewart** [comment]
I am working on a PR to which will hopefully improve documentation.  Targeting this weekend to have it raised.

**Comment by devinstewart** [comment]
If there is any addition feedback on PR #19 (Good or bad), I would be appreciative.  I am holding back on expanding this further (which it needs) until someone from the TSC approves.  Thank you!

**Comment by lloydbenson** [comment]
@mariusa does the new PR #23 address this issue?

**Comment by devinivy** [comment]
This was resolved by #23
